### PR TITLE
reuirements.txt file tested for linux system

### PR DESCRIPTION
**Pull request by saggu10417**
update requirements-linux,
removed pyAudio , 
removed pkg-resource from requirement , 
fix linux requirements file

### CODE REVIEW

**Comment by plusminuschirag** [comment]
Hi @saggu10417 It would be better if you remove the vext-gi file. It's inside the virtual-env and that's what should not be included in the Groot. Squeeze your commits into one also. Great Work.

**Comment by saggu10417** [comment]
Ya i will do it , Thanks .

**Comment by saggu10417** [comment]
Ya i will do it, Thanks

**Comment by saggu10417** [comment]
Ya, I will do it but I need your help. Can u please tell me how to squeeze
all the commits into one and it is done before pushing the changes or after?

**Comment by saggu10417** [comment]
Ya, I will do it but I need your help. Can u please tell me how to squeeze
all the commits into one and it is done before pushing the changes or after?

**Comment by sarthak247** [comment]
> Ya, I will do it but I need your help. Can u please tell me how to squeeze
> all the commits into one and it is done before pushing the changes or after?

There's a link in the README which describes how to squash commits into one

File: requirements/requirements-linux.txt [hunk 1]
```diff
@@ -12,7 +12,6 @@ lazy-object-proxy==1.3.1
 lxml==4.3.2
 mccabe==0.6.1
 playsound==1.2.2
-PyAudio==0.2.11
```
Comment (omi10859) [inline requirements/requirements-linux.txt:17]: we need PyAudio to play sound

**Review by omi10859 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**